_includes/alumni.html

```diff
@@ -5,7 +5,7 @@
       <div class="mod modSectionHeader ">
         <div class="special-title centered-text separateline">
           <a name="alumni"></a>
-          <h2 class="separate" style="color: ;">Alumni</h2>
+          <h2 class="separate" style="color: black;">Alumni</h2>
```

Comment:
i don't think the headers are black. Just remove the style attribute here.

Reply:
fixed